### PR TITLE
Dockerise

### PR DESCRIPTION
**Pull request by OscarVanL**
Dockerise measurement bot for rapid deployment, as the world economy would surely collapse without a reliable measurement system.

uWSGI.ini configuration will be left as an exercise for the reader

### CODE REVIEW

File: data/lengths.csv [hunk 1]
```diff
@@ -78,5 +78,6 @@ Unit,Value
 "the diameter of Saturn's rings", 250000000
 "the estimated sea level rise if the entire Antarctic ice sheet melted", 58.3
 "the supposed height of Disney Frozen's Olaf", 1.6256
-"the worlds longest conga line", 5.47
+"the worlds longest conga line", 25000
 "the height of the largest confirmed moose", 2.33
+"the giant space napkin", 384400000
```
Comment (jacobpennington821) [inline data/lengths.csv:83]: I feel this should be `the length of a giant space napkin, if a giant space napkin were, on average, to stretch from the surface of the earth to the surface of the moon`

File: main.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from routes import app
+
+app.config["VERSION"] = "0.1.2"
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=80)
```
Comment (jacobpennington821) [inline main.py:6]: Should it be port 80? Will this allow it to run on https?

**Review by jacobpennington821 — CHANGES_REQUESTED** [review]
Please help me I'm stuck in a github code review.

File: main.py (hunk 2 above)
Comment (OscarVanL) [reply main.py:6]: fuck knows tbf fam
Also how do u do Ur Https certs in your existing configuration?

**Pull request closed without merging**